flyteplugins/go/tasks/plugins/array/outputs_test.go

```diff
@@ -353,6 +353,7 @@ func TestAssembleFinalOutputs(t *testing.T) {
 		ow := &mocks2.OutputWriter{}
 		ow.OnGetOutputPrefixPath().Return("/prefix/")
 		ow.OnGetOutputPath().Return("/prefix/outputs.pb")
+		ow.OnGetErrorPath().Return("/location/prefix/error.pb")
```

Comment:
Tiny nit (non blocking):
Why `/location` before `/prefix` like in the line above? Currently the `error.pb` and the `outputs.pb` are in the same bucket uri.

Reply:
Sure, let me fix that. It was copied from some other example.

Reply:
Fixed